test/types.js

```diff
@@ -330,3 +330,4 @@ export const json = T.fn([T.any], T_isomorphism)
 // Auxiliary
 
 export const seemsArrayLike = T.fn([T.any], T.boolean)
+export const pointer = T.fn([T.string], T_lens)
```

Comment:
I've tried to keep the "sections" in the docs, the main implementation file and the types file in the same order/structure.  IOW, `pointer` would go under "Interop" section like in the docs.

Reply:
I'm on it.